src/patternfly/components/InlineEdit/examples/InlineEdit.md

```diff
@@ -0,0 +1,352 @@
+                {{/button}}
+              {{/inline-edit-action}}
+            {{/inline-edit-action}}
+            {{#> inline-edit-action}}
+              {{#> dropdown id=(concat table--id "-dropdown-kebab-right-aligned-1") dropdown--IsActionMenu="true" dropdown-menu--modifier="pf-m-align-right" dropdown-toggle--modifier="pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}
+              {{/dropdown}}
+            {{/inline-edit-action}}
+          {{/inline-edit-group}}
+        {{/table-td}}
+      {{/table-tr}}
+    {{/table-tbody}}
+  {{/table}}
+{{/inline-edit}}
+```
+
+## Documentation
+
+### Accessibility
+All accessibility requirements for inputs apply to elements within inline edit.
+
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `id` | `[labelling element]` | Provides a reference for toggle button and/or editable content. |
+| `aria-label="descriptive content"` | `.pf-c-inline-edit__toggle > button` | Provides an accessible name for toggle button. **Required** |
+| `aria-labelledby="[labelling element] [toggle button id]"` | `.pf-c-inline-edit__toggle > button` | Provides an accessible description for toggle button. **Required** |
+
+### Usage
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-inline-edit` | `<form>`, `<div>` | Initiates the inline edit component. **Required** |
+| `.pf-c-inline-edit__value` | `*` | Initiates an inline edit value. **Required** |
+| `.pf-c-inline-edit__input` | `*` | Initiates an inline edit input. **Required** |
+| `.pf-c-inline-edit__action` | `*` | Initiates an inline edit action (visible when inline edit region is active). **Required** |
+| `.pf-c-inline-edit__value-group` | `*` | Initiates an inline edit value group. |
+| `.pf-c-inline-edit__input-group` | `*` | Initiates an inline edit input group. This group can be nested (check/radio group). |
+| `.pf-c-inline-edit__action-group` | `*` | Initiates an inline edit action group that wraps action buttons within active, inline edit regions. |
+| `.pf-m-editable` | `.pf-c-inline-edit`, `.pf-c-inline-edit [block level element]` | Modifies an inline edit region for editable state. |
+| `.pf-m-edit` | `.pf-c-inline-edit__action` | Modifies for the edit action state. |
+| `.pf-m-save` | `.pf-c-inline-edit__action` | Modifies for the save action state. |
+| `.pf-m-cancel` | `.pf-c-inline-edit__action` | Modifies for the cancel action state. |
+| `.pf-m-valid` | `.pf-c-inline-edit__action.pf-m-save` | Modifies the save button state. |
+| `.pf-m-icon-group` | `.pf-c-inline-edit__action-group` | Modifies an inline edit action group item spacing. |
+| `.pf-m-inline` | `.pf-c-inline-edit__action-group` | Modifies an inline edit action group item spacing. |
+| `.pf-m-enable` | `.pf-c-inline-edit__action-group` | Modifies an inline edit action group visibility. |
+| `.pf-m-confirm` | `.pf-c-inline-edit__action-group` | Modifies an inline edit action group visibility. |
```

Comment:
This has the same description as `pf-m-enable`

Reply:
That's correct

Reply:
The classes override the default state for `.pf-m-action-group`, specifically for the table example. There's a group of buttons that enable edit mode, and another set that confirms changes whose visibility has to be toggled.

Reply:
So should one be used over another? @mattnolting